packages/hint-http-compression/tests/_tests.ts

```diff
@@ -268,23 +263,20 @@ const testsForBrotli: HintTest[] = [
         })
     },
     {
-        after() {
-            (utils as any).asyncTry = originalAsyncTry;
-        },
-        before() {
-            (utils as any).asyncTry = (fetch: IFetchFunction) => {
-                return (target: string, headers: HttpHeaders) => {
-                    if (!target || !target.includes('script.js') || headers['Accept-Encoding'] !== 'br') {
-                        return fetch(target, headers);
-                    }
-
-                    return null;
-                };
-            };
+        name: `If a request throws an exception, if should be managed and report an error (brotli)`,
+        overrides: {
+            '@hint/utils': {
+                asyncTry(fetch: IFetchFunction) {
```

Comment:
If we want to get really fancy I may be able to use generics in the type definition of `overrides` to derive this type automatically based on the referenced module name. Which would be cool since it'd flag typos or signature mis-matches versus the actual module. I feel it's out of scope here since we're trying to unblock a dependency update, but could be neat to explore in the future.

Reply:
Yep, that would be cool. Maybe you can open an issue to track that :)

Reply:
Done!